lib/IAMClient.js

```diff
@@ -52,6 +53,19 @@ class IAMClient {
         }
         this.accessKey = accessKey;
         this.secretKeyValue = secretKeyValue;
+        this.log = new Logger('IAMClient', {
+            level: 'info',
```

Comment:
These levels should be set by config. If config is not available then it can default to info and error.